uptane-standard.md

```diff
@@ -273,7 +273,7 @@ The following use cases provide a number of scenarios illustrating the manner in
 An OEM plans to install Uptane on new vehicles. This entails the following components: code to perform full and partial verification, the latest copy of the relevant metadata, the public keys, and an accurate attestation of the latest time. The OEM then either requires its tier-1 suppliers to provide these materials to the suppliers' assembly lines or can choose to add the materials later at the OEM's assembly lines. The OEM's implementation is Uptane-compliant if:
 
 1. all primaries perform full verification;
-1. all secondaries that are updated via OTA perform partial verification at a minimum; and
+1. all secondaries that are updated via OTA MUST at least perform partial verification; and
```

Comment:
Normally "Primary" and "Secondary" should be capitalized according to our style guide. I must've missed these because they are plural. Anyway, not critical and not the fault of your PR.

Reply:
We can't add a MUST until 2.0, but I applied the suggestion without the MUST in it.